docs/topics/testing/tools.txt

```diff
@@ -226,6 +226,11 @@ Use the ``django.test.Client`` class to make requests.
             you can call :func:`json.dumps` on ``data`` before passing it to
             ``post()`` to achieve the same thing.
 
+        .. versionchanged:: 2.2
+
+            The JSON serialization was extended to support all types serializable
```

Comment:
This note should be "self-contained" (i.e. the main paragraph above should reflect the new behavior and this note should say something to the effect of "In older versions, only data dictionaries are serialized."

Reply:
384455f1e2